src/Type/PostObject/Connection/PostObjectConnectionArgs.php

```diff
@@ -243,6 +243,13 @@ private static function fields( $connection ) {
 					'type' => Types::post_status_enum(),
 				],
 
+				/**
+				 * List of post status parameters
+				 */
+				'stati' => [
```

Comment:
I understand the logic here to mirror WP_Query functionality. I missed the initial PR; my only question is whether we considered consolidating on a single arg that always takes an array of statuses, even if you are only querying one, e.g.:

```
status: ['draft']
```

```
status: ['draft', 'publish']
```

Reply:
@chriszarate right now the `status` arg accepts an individual status and this introduces a plural `stati` which accepts a list, and does so in a non-breaking fashion.

Reply:
Makes sense, just raising opportunities for consolidation. Glad to have this functionality!